src/main/java/uk/gov/register/presentation/view/ItemView.java

```diff
@@ -26,4 +28,13 @@
     public Map<String, FieldValue> getContent() {
         return item.getFieldsStream().collect(Collectors.toMap(Map.Entry::getKey, itemConverter::convert));
     }
+
+    @Override
+    public CsvSchema csvSchema() {
+        CsvSchema.Builder schemaBuilder = new CsvSchema.Builder() ;
+        for (String value : getRegister().getFields()) {
```

Comment:
@philandstuff this is now using register fields as headers in building the schema